src/consolelogger.c

```diff
@@ -203,16 +203,29 @@ void consolelogger_log(LOG_CATEGORY log_category, const char* file, const char*
     time_t t;
     va_list args;
     va_start(args, format);
+#if (defined(_MSC_VER) && defined(_DEBUG))
+    char* debugPrint;
+#endif
 
     t = time(NULL);
 
     switch (log_category)
     {
     case AZ_LOG_INFO:
         (void)printf("Info: ");
+#if (defined(_MSC_VER) && defined(_DEBUG))
+        OutputDebugStringA("Info: ");
+#endif
         break;
     case AZ_LOG_ERROR:
         (void)printf("Error: Time:%.24s File:%s Func:%s Line:%d ", ctime(&t), file, func, line);
+#if (defined(_MSC_VER) && defined(_DEBUG))
+        if ((debugPrint = printf_alloc("Error: Time:%.24s File:%s Func:%s Line:%d ", ctime(&t), file, func, line)) != NULL)
+        {
+            OutputDebugStringA(debugPrint);
+            free(debugPrint);
+        }
```

Comment:
> } [](start = 7, length = 2)

At least would be nice to have the branch with a comment in it that we're explicitly not going to report the error.

Reply:
I assume you mean the allocation error: good point: I think I'll actually print something to show that we failed alloc.

Reply:
Yep, that's what I meant :-). Cheers!